rules/windows/process_creation/win_susp_prog_location_process_starts.yml

```diff
@@ -14,14 +14,14 @@ logsource:
     product: windows
 detection:
     selection:
-        Image:
-            - '*\$Recycle.bin'
-            - '*\Users\Public\\*'
-            - 'C:\Perflogs\\*'
-            - '*\Windows\Fonts\\*'
-            - '*\Windows\IME\\*'
-            - '*\Windows\addins\\*'
-            - '*\Windows\debug\\*'
+        Image|contains:
+            - '\$Recycle.bin'
```

Comment:
it's `startswith` for `\$Recycle.bin'`

Reply:
Fixed with endswith for recycle.bin and startswith for C:\Perflogs\